droidlet/memory/craftassist/mc_memory.py

```diff
@@ -28,6 +28,7 @@
     SchematicNode,
     NODELIST,
 )
+from droidlet.perception.craftassist.heuristic_perception import check_inside
```

Comment:
noooo can we not have memory import from perception plz ?

Reply:
so are you sure this is bad?  the perception is functional, and we are using it as a function directly on memories.  essentially its behaving the same as counting the number of blocks in a block object, etc.- it only sees the memory node it is operating on (and not the world directly).  One option is just to put it memory altogether for this case, we only use check_inside and find_inside on memory nodes.  We could alternatively have an "on_demand_perception" mediated by the agent, but would need to decide the interface for that